sqlite3/assets/wasm/bridge.h

```diff
@@ -50,10 +50,12 @@ import_dart("function_xInverse") extern void dartXInverse(
 import_dart("function_xFinal") extern void dartXFinal(sqlite3_context *ctx);
 import_dart("function_xValue") extern void dartXValue(sqlite3_context *ctx);
 import_dart("function_forget") extern void dartForgetAboutFunction(void *ptr);
-import_dart("function_hook") extern void dartUpdateHook(void *id, int kind,
-                                                        const char *db,
-                                                        const char *table,
-                                                        sqlite3_int64 rowid);
+import_dart("function_update_hook") extern void dartUpdateHook(void *id, int kind,
```

Comment:
Please keep the existing name unchanged, newer versions of `package:sqlite3` need to be compatible with older WebAssembly bundles too.